doc/api/stream.md

```diff
@@ -2077,6 +2077,27 @@ Calling `Readable.from(string)` or `Readable.from(buffer)` will not have
 the strings or buffers be iterated to match the other streams semantics
 for performance reasons.
 
+### `stream.Readable.concat(streams, [options])`
+
+* `streams` {Readable[]|Iterable[]|AsyncIterable[]} List of readable streams or iterator of readable stream generator
```

Comment:
```suggestion
* `streams` {Readable[]|Iterable[]|AsyncIterable[]} List of readable streams or
iterator of readable stream generator.
```
Every line must be 80 characters in length and this line should end with a period.